app/views/home/_users.html.erb

```diff
@@ -0,0 +1,119 @@
+<h1 class="center-heading"><%= I18n.t('dashboard.user_list') %></h1>
```

Comment:
Can this file be deleted?

Reply:
I think it is still used because the filtering is not implemented yet